src/pages/_app.js

```
@@ -1,5 +1,17 @@
+import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
+import Layout from "@/components/layout";
+
+const queryClient = new QueryClient();
 
 export default function App({ Component, pageProps }) {
-  return <Component {...pageProps} />;
+  return (
+    <QueryClientProvider client={queryClient}>
+      <Layout>
+        <Component {...pageProps} />;
+      </Layout>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)